foam_controlwrapper/foam_files.py

```diff
@@ -80,14 +92,314 @@ def writeFoamFiles(self, caseDirectory, solver):
 
         """
 
-        self.createDirectories(caseDirectory)
-        fileContent = self.createFileContent(solver)
+        self.create_directories(caseDirectory)
+        fileContent = self.create_file_content(caseDirectory, solver,
+                                               time, writeFields)
         for file in fileContent:
             try:
-                f = open(os.path.join(caseDirectory, file), 'w')
+                full_name = os.path.join(caseDirectory, file)
+                f = open(full_name, 'w')
                 f.write(fileContent[file])
                 f.close()
             except IOError:
                 error_str = "Can't write file: {}"
                 raise ValueError(error_str.format(os.path.join(caseDirectory,
                                                                file)))
+
+    def modify_files(self, case, SP, BC, solver, SPExt):
+
+        dire = SolutionDirectory(case, archive="SimPhoNy")
+        dire.clearResults()
+
+#        startTime = SP["startTime"]
+        startTime = 0
+        nOfTimeSteps = SP[CUBA.NUMBER_OF_TIME_STEPS]
+        deltaT = SP[CUBA.TIME_STEP]
+        endTime = nOfTimeSteps*deltaT
+        writeInterval = endTime-startTime
+        # if empty type boundary condition is used this must be
+        # changed in foam's polyMesh/boundary file to the same
+        # type (default type is patch)
+        pressureBCs = BC[CUBA.PRESSURE]
+        emptyBoundaries = []
+        for boundary in pressureBCs:
+            if pressureBCs[boundary] == "empty":
+                emptyBoundaries.append(boundary)
+
+        boundaryFile = os.path.join(case, 'constant', 'polyMesh', 'boundary')
+        try:
+            control = ParsedParameterFile(boundaryFile, boundaryDict=True)
+            boundaries = control.content
+            for boundary in emptyBoundaries:
+                for bi in range(len(boundaries)):
+                    filb = boundaries[bi]
+                    if filb == boundary:
+                        boundaries[bi+1]['type'] = "empty"
+                        break
+
+        except IOError:
```

Comment:
Please she comment https://github.com/simphony/simphony-openfoam/pull/10/files#r26569208. These is no need for the try...except block.